### DISCUSSION TITLE
Feature Request: Implement a Citation System Similar to LaTeX, Academic Articles, and MediaWiki

### DISCUSSION THREAD

**Discussion by IamTaoChen** [outline/outline#5357] [Ideas]
Dear Outline Developers,

As a regular user of Outline, I am continually impressed by the platform's simplicity and utility. However, I've identified an area for potential improvement that would greatly enhance the user experience, especially for those of us who often produce academic or research-oriented documents.

At present, Outline, based on Markdown, does not inherently support a citation or referencing system similar to those found in LaTeX, academic articles, or MediaWiki. In academic and research contexts, the ability to insert numbered citations within the text and automatically generate a corresponding list of references at the end of the document is crucial. This not only improves readability but also allows for easier cross-referencing.

Consequently, I would like to propose a feature request for the implementation of a citation system similar to what is used in LaTeX, academic articles, or MediaWiki. This feature would enable users to include numbered citations within their text and then automatically create a corresponding reference list at the end of the document. For instance, inserting [1] in the text would generate a reference numbered [1] in the reference list at the end of the document.

I believe this feature would be highly beneficial for users who regularly need to cite sources in their documents, and it could potentially attract a new audience from the academic and research communities. I appreciate your consideration of this request, and I eagerly await potential future developments in this area.

Best regards
<img width="800" alt="image" src="https://github.com/outline/outline/assets/42793494/b17c3b27-f70f-4987-bfcc-d6e653b56533">

**Comment by michael-rowe** [comment]
I'm looking at wiki options for a small community of health professions educators interested in building a knowledge base that's more dynamic than a basic wiki. And Outline looks like a great option. However, without citation support (even if only as numbered footnotes), I know that my colleagues are going to have reservations.
